7339 ol configured tai (#7384)
OpenLiberty/docs: modules/ROOT/pages/troubleshooting.adoc

This error occurs when a keystore element exists in the configuration without an ID field.
If you use a minimal TLS configuration, set the `ID` field to `defaultKeyStore`.

[#Troubleshooting_TAI]
== Troubleshooting Trust Association Interceptor

> To avoid potential issues, users must not configure the `TrustAssociationInterceptor` component to call the `InitialDirContext` class with the Java property `java.naming.ldap.factory.socket` set to a socket factory other than Liberty's own (`com.ibm.ws.ssl.protocol.LibertySSLSocketFactory`). Setting this property to other factories can cause a `NoClassDefFoundException`.

Reviewer: ```suggestion
When you configure the `TrustAssociationInterceptor` component to call the `InitialDirContext` class, the `java.naming.ldap.factory.socket` property must be set to the`com.ibm.ws.ssl.protocol.LibertySSLSocketFactory` Liberty socket factory. Setting this property to other factories can cause a `NoClassDefFoundException`.
```
